### PR TITLE
add up- and downvote functionallity

### LINKED ISSUE #36: Add up- and downvote actions / logic inside PollListItem component
At this moment, the up- and downvote action is only prepared, not implemented. Add inside `upvote` and `downvote` an async call to the backend api.

### PR DESCRIPTION
**Pull request by martincodes-de**
this pr adds up and downvote functionallity to poll list item
 closes #36

### CODE REVIEW

**Comment by NiklasBGit** [comment]
I think I added this by accident in #26 so this one can be closed?

File: components/PollListItem.vue [hunk 1]
```diff
@@ -91,11 +98,22 @@
       },
 
       async upvote() {
+        await useFetch("https://backberry.ddev.site/api/votes/create", {
+          method: 'POST',
+          body: { description: true, pollId: this.id }
```
Comment (GrafBunsen) [inline components/PollListItem.vue:103]:
```suggestion
          body: { description: true, pollId: this.id },

```

**Review by GrafBunsen — CHANGES_REQUESTED** [review]
upvote count is not updated when clicking, new value only loads after refresh (on index page)

**Comment by GrafBunsen** [comment]
dont merge, feature might be obsolete

**Comment by GrafBunsen** [comment]
see comment above

**Pull request closed without merging**